controllers/auth_controller.go

```diff
@@ -38,11 +42,17 @@ func (ctrl *authController) Register(context *gin.Context) {
 	coll := mgm.Coll(&body)
 
 	// Ensure that this email hasn't been registered already.
-	err = coll.First(bson.M{"email_address": body.EmailAddress}, &body)
+	err = coll.First(bson.M{"emailAddress": body.EmailAddress}, &body)
 	if err == nil {
 		context.AbortWithStatusJSON(http.StatusBadRequest, gin.H{"error": "User with email already exists."})
 		return
 	}
+	// Ensure that this username hasn't been registered already.
+	err = coll.First(bson.M{"username": body.Username}, &body)
```

Comment:
Not technically part of this ticket (https://github.com/drift-org/backend/issues/16), but good for doing it! link this issue (edit ur description and add "closes #16") so it closes.

Reply:
hmm maybe not. you didn't add the lowercase conversion. Maybe quickly add that or don't add the "closes" keyword.